doc/old_and_new_proposal.md

```diff
@@ -217,6 +217,7 @@ Besides these, there is another element:
   * `desired_size` *(disksize, default: `0 B`)*
   * `min_size` *(disksize, default: `0 B`)*
   * `max_size` *(disksize, default: `unlimited`)*
+  * `max_size_limit` *(disksize, default: `unlimited`)*
```

Comment:
I guess it is not needed to make this value configurable via control file. It is used only for prep partitions, and that partitions are automatically proposed (not configurable in control file at all).

Reply:
It's currently in a state of limbo anyway - `control.rng ` would need to be adjusted. I'm fine with removing it from `FEATURES` in `VolumeSpecification` - I just thought it consistent to have all attributes configurable in this class.

Reply:
For me looks fine to have all attributes in `VolumeSpecification`, but not all volume specification attributes have to come from the control file. So, I would simply remove it from the documentation about the new settings.